fluxbox.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: fluxbox
+  version: 1.3.7
+  epoch: 0
+  description: Fluxbox Window Manager
+  copyright:
+    - license: GPL-2.0-or-later
+
+environment:
+  contents:
+    packages:
+      - autoconf
+      - automake
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - gettext-dev
+      - libtool
+      - libx11-dev
+      - libxext-dev
+      - pkgconf-dev
+
```

Comment:
https://git.alpinelinux.org/aports/tree/community/fluxbox/APKBUILD#n11

AFAICS, `imlib2` also required which we need to build `libid3tag` first. I was about to get into it 🙈

Reply:
Sure, feel free to use this as reference material in that case (marked as draft for now). there where no complication errors, is imlib2 needed as a runtime dependency?

Reply:
I think its required as build-dependency but since pipeline is green, we can say its not a hard-requirement. But it would be difficult to predict the runtime behaviour without having this dep, but i will package `imlib2` anyway since will be needed by other deps. I got this 👍